Sources/Screens/Tokens/TokensViewController.swift

```diff
@@ -13,7 +14,8 @@ class TokensViewController: UIViewController {
         }
     }
     
-    init() {
```

Comment:
Might be nice to have the parameter name just for clarity & consistence if ya don't mind